.gitignore

```diff
@@ -31,6 +31,7 @@ make.inc
 examples/*
 !examples/*/input.py
 !examples/*/*/*.ipynb
+!examples/*/*/*/*.ipynb
```

Comment:
Probably we should do `!examples/**/*.ipynb`, as a concise and more comprehensive way?